$NetBSD: patch-ac,v 1.6 2010/04/13 06:53:20 obache Exp $

Avoid this:

/usr/pkgsrc/graphics/pstoedit/work/.buildlink/include/ImageMagick/magick/magick-type.h:120: error: ISO C++ does not support 'long long'
/usr/pkgsrc/graphics/pstoedit/work/.buildlink/include/ImageMagick/magick/magick-type.h:121: error: ISO C++ does not support 'long long'

on alpha (and probably others)

--- configure.orig	2011-08-27 15:27:42.000000000 +0000
+++ configure
@@ -16859,7 +16859,7 @@ $as_echo "GNU on RiscOS" >&6; }
     { $as_echo "$as_me:${as_lineno-$LINENO}: result: GNU" >&5
 $as_echo "GNU" >&6; }
 #   need to disable long long warning here because of ImageMagick using it
-    SPECIAL_CXXFLAGS="-DBUGGYGPP -pedantic -Wall -Wwrite-strings -Wcast-qual -Wpointer-arith -Wno-long-long "
+    SPECIAL_CXXFLAGS="-DBUGGYGPP -Wall -Wwrite-strings -Wcast-qual -Wpointer-arith -Wno-long-long "
   fi
 elif test `echo "$CXX" | egrep -c 'xlC$'` -eq 1; then
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: AIX xlC" >&5
